cobbler/templar.py

```diff
@@ -22,35 +22,40 @@
 02110-1301  USA
 """
 
-import re
-import Cheetah
 import functools
 import os
 import os.path
 import pprint
+import re
+from typing import Optional, Union
+
+import Cheetah
+from typing.io import TextIO
```

Comment:
```suggestion
from typing import Optional, Union, TextIO

import Cheetah
```
Do you mean `typing` instead of `typing.io` here? According to the [docs](https://docs.python.org/3/library/typing.html#typing.TextIO) `TextIO` is from `typing`. The import could be simplified as suggested then.